core/src/js/components/accordion/accordion.js

```diff
@@ -0,0 +1,53 @@
+const accordions = document.querySelectorAll('.su-accordion');
+const titleButtons = document.querySelectorAll('.su-accordion__button');
+const expandButtons = document.querySelectorAll('.su-accordion__expand-all');
+const collapseButtons = document.querySelectorAll('.su-accordion__collapse-all');
```

Comment:
It is probably safe, but these are global queries and perhaps it might be better if the query was a descendant selector.

Reply:
Sure, will do.

Going to have less time today to do this but will push things bit by bit if I have time. Thank you for the review as usual!! 😄